velox/docs/functions/json.rst

```diff
@@ -2,6 +2,95 @@
 JSON Functions
 ==============
 
+JSON Format
+---------------
+JSON is a language-independent data format that represents data as
+human-readable text. A JSON text can represent a number, a boolean, a
+string, an array, an object, or a null, with slightly different grammar.
+For instance, a JSON text representing a string must escape all characters
+and enclose the string in double quotes, such as ``"123\n"``, whereas a JSON
+text representing a number does not need to, such as ``123``. A JSON text
+representing an array must enclose the array elements in square brackets,
+such as ``[1,2,3]``. More detailed grammar can be found on https://www.json.org.
+
+JSON Vectors
```

Comment:
I find the information in this section useful, but it is at a different level than the rest of the article. Here, we are talking to developers of the library rather than users. I wonder if there is a way to somehow highlight this distinction to avoid confusion.

Reply:
This is a good suggestion. I lift the subsection "JSON Vectors" to a section in parallel to the top-level "JSON Functions" section in the new version.